lib/engines/content_object_store/app/models/content_object_store/content_block_edition.rb

```diff
@@ -1,3 +1,5 @@
 class ContentObjectStore::ContentBlockEdition < ApplicationRecord
   include ContentObjectStore::Identifiable
+
+  accepts_nested_attributes_for :content_block_document
```

Comment:
Side question: how are we deciding what lives in this class vs in Identifiable? I can see we have block_type accessors in there but not sure what the boundary of that concern is anymore?

Reply:
Should `Identifiable` be limited to only copying behaviour/scope of existing `Identifiable`, or what would be shared between the two types of Editions? in which case, `block_type` should be lifted out, as that's unique to our Editions?

Reply:
I think `block_type` is OK in the `Identifiable` concern, as it's required for the relationship between the Edition and the Document. If that's the case, then this `accepts_nested_attributes_for` setup should be in here too I think

Reply:
In fact, once we have this in, the `block_type` and `title` stuff can be removed too

Reply:
> I find the name Identifiable actually a bit obscure in meaning and maybe that's causing the confusion?...

Yes this is where I'm at, I'm not clear what the meaning is which is half the issue. Is it everything to do with being associated with a Document perhaps, so`Documentable`? Once we're aligned it's easy to move this around.

Reply:
Yeah, that's how I read it, so maybe our naming is a bit off

Reply:
I suspect our naming is inherited from [the original Identifable class](https://github.com/alphagov/whitehall/blob/8822d03cb96fc36f69ec92db1a0c87ede55aee3c/app/models/concerns/edition/identifiable.rb#L1). Which itself seems to be doing more than identifying things. 

I propose we rename to `Documentable` or similar and move our changes into it? @Harriethw @pezholio 

That'll leave us diverging from Whitehall main which may make it harder to reuse from later, if we get there.

Reply:
Sounds like a plan 👍

Reply:
👍